src/main/resources/psw4j.properties

```diff
@@ -0,0 +1,9 @@
+### Argon2
```

Comment:
Can you add `global.banner=false` to remove the banner when the library is called